### PR TITLE
Fix in BatchNorm fusion

### PR DESCRIPTION
**Pull request by ermanok**

### CODE REVIEW

File: ai8x.py [hunk 1]
```diff
@@ -1473,7 +1473,7 @@ def _fuse_bn_layers(m):
 
                 r_mean = target_attr.bn.running_mean
                 r_var = target_attr.bn.running_var
-                r_std = torch.sqrt(r_var + 1e-20)
+                r_std = torch.rsqrt(r_var + target_attr.bn.eps)
```
Comment (MaximGorkem) [inline ai8x.py:1476]: It looks good.

Not an important issue but is there a specific reason why we use reciprocal sqrt here? Should we rename the variable to r_inv_std for clarity?

**Review by MaximGorkem — APPROVED** [review]
Looks good

File: ai8x.py (hunk 1 above)
Comment (ermanok) [reply ai8x.py]: reciprocal sqrt is used here as it is safer to handle division by very small numbers especially when gradient calculation is important. The name of the variable is changed as you suggest.

Comment (MaximGorkem) [reply ai8x.py]: Thanks!

**Review by MaximGorkem — APPROVED** [review]
Looks good, thanks

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**